README.md

```diff
@@ -14,61 +14,22 @@ React frontend for [HospitalRun](http://hospitalrun.io/): free software for deve
 
 ---
 
-**Version 1.0.0-beta is no longer supported. Version 2 is currently under development.**
+# Are you a user? If yes...
 
-- To contribute, follow the guidelines in the readme or alternatively ask for details on Slack channel [#contributors](https://hospitalrun-slack.herokuapp.com).
-- To use version 1.0.0-beta (not production ready) in a hospital facility, ask for support on Slack channel [#troubleshooting](https://hospitalrun-slack.herokuapp.com).
+[Visit this page for general information on the HospitalRun application](https://github.com/HospitalRun/hospitalrun/blob/master/README.md) including:
 
-<div align="center">
-
-[![Slack](https://img.shields.io/badge/Slack-Join%20our%20devs%20group-blueviolet?style=for-the-badge&logo=slack)](https://hospitalrun-slack.herokuapp.com)
-
-</div>
-
-# Staging area 
-
-You can follow developments by visiting the dedicated [staging environment](https://staging.hospitalrun.io). Use `username` / `password` as credentials to access.
-
-# Contributing
-
-Contributions are always welcome. Before contributing please read our [contributor guide](https://github.com/HospitalRun/hospitalrun-frontend/blob/master/.github/CONTRIBUTING.md).
-
-## Translation
-
-Use the standards in [this readme](https://github.com/HospitalRun/hospitalrun-frontend/tree/master/src/locales/README.md).
-
-## Online one-click setup for contributing
-
-Contribute to HospitalRun using a fully featured online development environment that will automatically: clone the repo, install the dependencies and start the webserver.
-
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/HospitalRun/hospitalrun-frontend)
-
-## Running Tests and Linter
+How can I deploy 1.0.0-beta? | Where do I report a bug or request a feature? | How can I contribute? (There are several other ways besides coding) | What is the project structure? | What is the application infrastructure? | Who is behind HospitalRun? etc.
```

Comment:
I think this should be a bulleted list to make it easier to read

Reply:
Done

Reply:
I don't see the changes 😀